Makefile

```
@@ -53,7 +52,8 @@ tests/serial: ALWAYS_RUN
 	@ shopt -s globstar ; ./test/bats/bin/bats $(BATS_FLAGS) test/$@/*.bats
 
 tests/parallel/set%: ALWAYS_RUN
-	@ shopt -s globstar ; ./test/bats/bin/bats $(BATS_FLAGS_PARALLEL) test/$@/**/*.bats
+	@ shopt -s globstar ; ./test/bats/bin/bats $(BATS_FLAGS) \
+		--no-parallelize-within-files --jobs $(BATS_PARALLEL_JOBS) test/$@/**/*.bats
```

Prefer vertically listing the explicit options, easier to grok and track changes :sweat_smile: 

```suggestion
	@ shopt -s globstar ; ./test/bats/bin/bats $(BATS_FLAGS) \
		--no-parallelize-within-files \
		--jobs $(BATS_PARALLEL_JOBS) \
		test/$@/**/*.bats
```

---

A reminder that `--no-parallelize-within-files` prevents `--jobs 1` for running a parallel set sequentially.

Although I wouldn't worry about this much as I'm doubtful the default of `--jobs 2` will cause anyone running these tests a problem, since we can still run a few individual tests with `make test/file1,file2,file3` easily enough.

---

If of interest to you:
- There is also a [tagging feature](https://bats-core.readthedocs.io/en/v1.8.2/writing-tests.html#tagging-tests) to be selective on what test cases to run in a file(s).
- `--no-parallelize-within-files` could be considered for removal if we can confidently evaluate which test files need to explicitly opt-out via [`export BATS_NO_PARALLELIZE_WITHIN_FILE=true` in `setup_file()`](https://bats-core.readthedocs.io/en/v1.8.2/usage.html#parallel-execution).

---

I have merged this to make CI pass again, but we can tackle this in another PR - I don't really mind since it's only a stylistic change :)